### PR TITLE
Fix: Display issue in grid outline [ED-11840]

### PR DESCRIPTION
**Pull request by murzin-nikita**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: assets/dev/scss/editor/preview/preview.scss [hunk 1]
```diff
@@ -48,8 +46,8 @@
 
 @mixin e-grid-outline-full-width {
 	@include e-grid-outline-base;
-	padding-right: var(--padding-right);
-	padding-left: var(--padding-left);
+	//padding-right: var(--padding-right);
```
Comment (hein-obox) [inline assets/dev/scss/editor/preview/preview.scss:49]: Please remove or uncomment.

File: assets/dev/scss/frontend/_container.scss [hunk 2]
```diff
@@ -134,8 +134,8 @@
 		width: 100%;
 		max-width: var(--content-width);
 		margin: 0 auto;
-		padding-inline-start: 0;
-		padding-inline-end: 0;
+		padding-inline-start: var(--padding-left, var(--padding-inline-start));
```
Comment (hein-obox) [inline assets/dev/scss/frontend/_container.scss:137]:
```suggestion
		padding-inline-start: var(--bc-padding-inline-start, var(--padding-inline-start));
```

File: assets/dev/scss/editor/preview/preview.scss [hunk 3]
```diff
@@ -92,6 +88,10 @@ html.elementor-html {
 
 	.e-grid-outline {
 		display: none;
+		padding-block-start: var(--padding-block-start);
```
Comment (hein-obox) [inline assets/dev/scss/editor/preview/preview.scss:91]:
```suggestion
		padding-block-start: var(--padding-top, var(--padding-block-start));
```

File: assets/dev/scss/editor/preview/preview.scss [hunk 4]
```diff
@@ -92,6 +88,10 @@ html.elementor-html {
...
+		padding-block-end: var(--padding-block-end);
```
Comment (hein-obox) [inline assets/dev/scss/editor/preview/preview.scss:92]:
```suggestion
		padding-block-end: var(--padding-bottom, var(--padding-block-end));
```

File: assets/dev/scss/editor/preview/preview.scss [hunk 5]
```diff
@@ -92,6 +88,10 @@ html.elementor-html {
...
+		padding-inline-start: var(--padding-inline-end);
```
Comment (hein-obox) [inline assets/dev/scss/editor/preview/preview.scss:93]:
```suggestion
		padding-inline-start: var(--bc-padding-inline-end, var(--padding-inline-end));
```

File: assets/dev/scss/editor/preview/preview.scss [hunk 6]
```diff
@@ -38,6 +35,7 @@
 	justify-content: var(--grid-justify-content);
 	align-content: var(--grid-align-content);
 	min-height: var(--min-height);
+	padding-block-start: var(--padding-block-start);
```
Comment (hein-obox) [inline assets/dev/scss/editor/preview/preview.scss:38]:
```suggestion
	padding-block-start: var(--padding-top, var(--padding-block-start));
```

File: assets/dev/scss/editor/preview/preview.scss [hunk 7]
```diff
@@ -26,6 +26,8 @@
 @mixin e-grid-outline-base {
 	padding-block-start: var(--bc-padding-block-start, var(--padding-block-start));
 	padding-block-end: var(--bc-padding-block-end, var(--padding-block-end));
+	padding-inline-start: var(--padding-inline-end);
+	padding-inline-end: var(--padding-inline-start);
```
Comment (hein-obox) [inline assets/dev/scss/editor/preview/preview.scss:30]: padding-inline-start: var(--bc-padding-inline-start, var(--padding-inline-start));
padding-inline-end: var(--bc-padding-inline-end, var(--padding-inline-end));

**Review by hein-obox — APPROVED** [review]

File: assets/dev/scss/frontend/_container.scss [hunk 8]
```diff
@@ -137,8 +137,8 @@
 		width: 100%;
 		max-width: var(--content-width);
 		margin: 0 auto;
-		padding-inline-start: 0;
-		padding-inline-end: 0;
```
Comment (Barcov) [inline assets/dev/scss/frontend/_container.scss:141]: @mykytamurzin Does this not change default padding inline-start from 0px to 10px?

```
	--padding-block-start: var(--container-default-padding-block-start, 10px);
	--padding-inline-end: var(--container-default-padding-inline-end, 10px);
	--padding-block-end: var(--container-default-padding-block-end, 10px);
	--padding-inline-start: var(--container-default-padding-inline-start, 10px);

**Pull request closed without merging**